### PR TITLE
[SITE] Improve the user experience of the site

### PR DESCRIPTION
**Pull request by lamber-ken**
### Motivation

Improve the user experience of the site.

image

### Changes

**Compare:**
https://lamber-ken.github.io/docs/4.10.0/getting-started/installation/
https://bookkeeper.apache.org/docs/4.10.0/getting-started/installation/

### CODE REVIEW

**Comment by lamber-ken** [comment]
Any suggestion are welcome 😃

**Comment by eolivelli** [comment]
@mino181295 PTAL

File: site/css/style.sass [hunk 1]
```diff
@@ -81,9 +81,9 @@ footer.footer
 .toc
   overflow-y: scroll
   bottom: 0
-  top: $navbar-height + 5rem
+  top: 50px
```
Comment (mino181295) [inline site/css/style.sass:84]: Change the $navbar-height variable instead

**Comment by mino181295** [comment]
Maybe it's a good idea to hide the right column when the with is 769~1200
immagine

Take a look at the **variables.sass and helpers.sass**

File: site/css/style.sass (hunk 1 above)
Comment (mino181295) [inline site/css/style.sass:84]: Change the $navbar-height variable instead

**Comment by lamber-ken** [comment]
Thanks @mino181295, let's me have a try.

File: site/css/style.sass (hunk 1 above)
Comment (lamber-ken) [reply site/css/style.sass]: Done.

**Comment by lamber-ken** [comment]
hi @mino181295 @eolivelli, I had tried several times. Compared to old site, I can improve the following three parts, so this pr is ready for merge, thanks.

image

**Comment by lamber-ken** [comment]
hi @mino181295 

> Maybe it's a good idea to hide the right column when the with is 769~1200

`variables.sass` and `helpers.sass` can not work.

**Comment by sijie** [comment]
I think the website was customized site. It is hard to maintain the site. I would suggest us migrating the documentation site to use https://docusaurus.io/ as what we do for pulsar.

**Comment by lamber-ken** [comment]
> I think the website was customized site. It is hard to maintain the site. I would suggest us migrating the documentation site to use https://docusaurus.io/ as what we do for pulsar.

 https://docusaurus.io/ Look nice, 👍

**Comment by eolivelli** [comment]
But if this fix helps in improving the website we can commit it as a hot fix.
It will take a lot of time to migrate to another system

**Comment by lamber-ken** [comment]
Agree with you all @eolivelli @sijie
- https://docusaurus.io/ Look nicer
- this pr can improve the user experience of the website in use.

**Comment by eolivelli** [comment]
@mino181295  are you ok with merging current patch in this form ?

 @sijie  we can switch to a new framework later
it looks like we do not have much energy these days in the community

**Comment by mino181295** [comment]
Yes, to me it's ok!

**Comment by Ghatage** [comment]
Awesome stuff @lamber-ken!
Also, if you're interested, we can start an email chain in dev to start scoping out the move to docusaurus.io

**Comment by lamber-ken** [comment]
> Awesome stuff @lamber-ken!
> Also, if you're interested, we can start an email chain in dev to start scoping out the move to docusaurus.io

Thanks, will check the mail list : )

**Comment by eolivelli** [comment]
This is a minor change and does not have impact in production code, I will commit it, no need for a second binding +1

**Pull request merged**